client/src/components/PinkHeader/PinkHeader.css

```diff
@@ -0,0 +1,63 @@
+*{
+    font-family: var(--default-font)
+}
```

Comment:
Can you delete this? if we need to set a style globally, we should do so in `index.css`. I don't think we want to set this globally though

Reply:
removed from `PInkHeader.css` and moved to `index.css` to see if we want to run it, else I'll comment it out.